charts/cert-manager-webhook-ovh/templates/deployment.yaml

```diff
@@ -69,6 +69,9 @@ spec:
           securityContext:
             allowPrivilegeEscalation: {{ .Values.securityContext.allowPrivilegeEscalation }}
             readOnlyRootFilesystem: {{ .Values.securityContext.readOnlyRootFilesystem }}
+            privileged: {{ .Values.securityContext.privileged }}
+            capabilities:
+{{ toYaml .Values.securityContext.capabilities | indent 14 }}
```

Comment:
What happens if the `capabilities` aren't specified?

Reply:
As the pod should not need any capabilities, we may hardcode the ```drop: [ALL]```

Reply:
This explicit drop is [required by kubeaudit capabilities auditor](https://github.com/Shopify/kubeaudit/blob/main/docs/auditors/capabilities.md#explanation)